Dockerfile.downstream

```diff
@@ -1,7 +1,7 @@
 ARG BUILDVERSION
 
 # Build the manager binary
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23 as builder
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.24 as builder
```

Comment:
I don't think that exists at the moment (?)

Reply:
how do you check that ? :thinking:

Reply:
Sorry I missed that comment!
I'm just doing:

```
podman pull --authfile=/path/to/pull-secret.json brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.24
```

btw, it's now available :-)